spec-0007/index.md

```diff
@@ -5,69 +5,69 @@ author:
+are more broadly applicable. Use of random number generators other than those provided
+by NumPy are not considered at this time, but may be addressed in a future version.
 
 ### Concepts
 
-- `BitGenerator`: Generates a stream of pseudo-random bits. The default generator in NumPy (`np.random.default_rng`) uses PCG64.
+- `BitGenerator`: Generates a stream of pseudo-random bits. The default generator in NumPy (`numpy.random.default_rng`) uses PCG64.
 - `Generator`: Derives pseudo-random numbers from the bits produced by a `BitGenerator`.
 - `RandomState`: a [legacy object in NumPy](https://numpy.org/doc/stable/reference/random/index.html), similar to `Generator`, that produces random numbers based on the Mersenne Twister.
 
 ### Constraints
 
 NumPy, SciPy, scikit-learn, scikit-image, and NetworkX all implement pseudo-random seeding in slightly different ways.
 Common keyword arguments include `random_state` and `seed`.
-In practice, the seed is unfortunately also often controlled using `np.random.seed`.
+In practice, the seed is unfortunately also often controlled using `numpy.random.seed`.
 
 ## Implementation
 
-<!--
-Discuss how this would be implemented.
--->
-
 Legacy behavior in packages such as scikit-learn (`sklearn.utils.check_random_state`) typically handle `None` (use the global seed state), an int (convert to `RandomState`), or `RandomState` object.
 
 Two strong motivations for moving over to `Generator`s are:
 
 (1) they avoid naïve seeding strategies, such as using successive integers, via the underlying [SeedSequence](https://numpy.org/doc/stable/reference/random/parallel.html#seedsequence-spawning);
-(2) they avoid using global state (from `np.random.mtrand._rand`).
+(2) they avoid using global state (from `numpy.random.mtrand._rand`).
 
 Our recommendation here is a deprecation strategy which does not in _all_ cases adhere to the Hinsen[^hinsen] principle.
 
 The [deprecation strategy](https://github.com/scientific-python/specs/pull/180#issuecomment-1515248009) is:
 
-1. Accept both `rng` and `random_state` keyword arguments.
-2. If `rng=None`, handle `random_state` as in legacy behavior (see above), except use a compatible Generator instead of RandomState.
+1. Accept both `rng` and `random_state`/`seed` keyword arguments.
+2. If `rng=None`, handle `random_state`/`seed` as in legacy behavior (see above), except use a compatible Generator instead of RandomState.
    A DeprecationWarning is raised to warn about a future change in behavior.
-3. After <X time>, use only `rng`, seeding with `default_rng(rng)`.
-   Raise an error if `random_state` is provided.
-4. At a time of the library's choosing, remove any machinery related to `random_state`.
+3. After <X time>, use only `rng`, validated with `numpy.random.default_rng`.
+   Raise an error if `random_state`/`seed` is provided.
+4. At a time of the library's choosing, remove any machinery related to `random_state`/`seed`.
```

Comment:
After the deprecation period, any machinery related to `random_state`/`seed` is no longer used, so I think it's implicit that it can be removed.
```suggestion
```
In any case, this goes a bit beyond the deprecation strategy, and this section is longer than it was, so might as well try this.